var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -606,3 +606,8 @@ def build_amd(self):
     def install_test(self):
         with working_dir("test"):
             python("run_test.py")
+
+    @property
+    def cmake_prefix_paths(self):
+        cmake_prefix_paths = [join_path(python_platlib, "torch", "share", "cmake")]
```

Comment:
This now fails with:
```console
==> Installing libtorch-example-app-1.1-cf6y5hiyqyhileyzd6hdew7tidsztqfp
==> No binary for libtorch-example-app-1.1-cf6y5hiyqyhileyzd6hdew7tidsztqfp found: installing from source
==> Error: NameError: name 'python_platlib' is not defined

/home/wdconinc/git/spack/var/spack/repos/builtin/packages/py-torch/package.py:613, in cmake_prefix_paths:
        611    def cmake_prefix_paths(self):
        612        cmake_prefix_paths = [
  >>    613            join_path(python_platlib, "torch", "share", "cmake")
        614        ]
        615        return cmake_prefix_paths
```
No `self.python_platlib` either. 

I'd replace this again with:
```suggestion
        cmake_prefix_paths = [join_path(self.prefix, self.spec["python"].package.platlib, "torch", "share", "cmake")]
```
unless you have another suggestion? It seems there is maybe some way to call `getmodule` which has `platlib` but I couldn't figure it out yet.

Reply:
Yes, let's go back to the way you had it before (with torch/share/cmake split like above).